bluetooth_battery.py

```diff
@@ -61,23 +66,34 @@ def getATCommand(sock, line, device):
 
     return True
 
+
 def find_rfcomm_port(device):
-    uuid="0000111e-0000-1000-8000-00805f9b34fb"
+    """
+    Find the RFCOMM port number for a given bluetooth device
+    """
+    uuid = "0000111e-0000-1000-8000-00805f9b34fb"
     proto = bluetooth.find_service(address=device, uuid=uuid)
     if len(proto) == 0:
         print("Couldn't find the RFCOMM port number")
         return 4
-    else:
-        for j in range(len(proto)):
-            if 'protocol' in proto[j] and proto[j]['protocol'] == 'RFCOMM':
-                port = proto[j]['port']
-                return port
+
+    for j, _ in enumerate(proto):
+        if 'protocol' in proto[j] and proto[j]['protocol'] == 'RFCOMM':
+            port = proto[j]['port']
+            return port
+    return -1
```

Comment:
I have a suggestion for this. I think you can just use a regular old for loop without `enumerate`.
And also return a positive default value as port number can't be negative.
```suggestion
    for pr in proto:
        if 'protocol' in pr and pr['protocol'] == 'RFCOMM':
            port = pr['port']
            return port
    return 4
```